internal/pkg/cli/cmd/template/repo/cmd.go

```diff
@@ -0,0 +1,21 @@
+package repo
+
+import (
+	"github.com/spf13/cobra"
+
+	"github.com/keboola/keboola-as-code/internal/pkg/cli/dependencies"
+	"github.com/keboola/keboola-as-code/internal/pkg/cli/helpmsg"
+)
+
+func Commands(d dependencies.Provider) *cobra.Command {
+	cmd := &cobra.Command{
+		Use:  `repo`,
```

Comment:
S tou zkratkou si nejsem úplně jistej. Chápu že to má nejspíš uspořit psaní (i když i to stejně vesměs řeší napovídání), ale přijde mi že to potom je trochu na vrub čitelnosti. Co myslíš?

Reply:
ok, dame `repository`